src/components/codeContext.tsx

```diff
@@ -61,7 +61,13 @@ const DEFAULTS: CodeKeywords = {
   ],
 };
 
-const CodeContext = React.createContext(DEFAULTS);
+type CodeContextType = {
+  codeKeywords: CodeKeywords;
+  sharedCodeSelection: any;
+  sharedKeywordSelection: any;
+};
+
+const CodeContext = React.createContext(null as CodeContextType | null);
```

Comment:
Does this work as 

```
const CodeContext = React.createContext<CodeContextType | null>(null);

```

Reply:
Yeah it probably would of - I forgot (or didnt know) about that syntax until 50% of the way through this.